pir/pir-internal/src/main/java/com/duckduckgo/pir/internal/pixels/PirPixel.kt

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright (c) 2025 DuckDuckGo
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.duckduckgo.pir.internal.pixels
+
+import com.duckduckgo.app.statistics.pixels.Pixel.PixelType
+import com.duckduckgo.app.statistics.pixels.Pixel.PixelType.Count
+import com.duckduckgo.app.statistics.pixels.Pixel.PixelType.Daily
+import com.duckduckgo.app.statistics.pixels.Pixel.PixelType.Unique
+
+enum class PirPixel(
+    val baseName: String,
+    private val types: Set<PixelType>,
+    private val withSuffix: Boolean = true,
+) {
+    PIR_INTERNAL_MANUAL_SCAN_STARTED(
+        baseName = "m_pir-internal_manual-scan_started",
```

Comment:
nit: I remember coms conversations about not using the `m_` prefix anymore for pixels. We don't use them in other products/features, but can't remember if we made the decision to stop using it or not.

Reply:
Updated. Thank you! Missed that comms completely.